cli_tools/gce_onestep_image_import/README.md

```diff
@@ -30,28 +28,29 @@ go get github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_onestep_
   windows-2016-byol, windows-7-byol.
   
 To import from AWS, all of these must be specified:
-+ `-aws_access_key_id=AWS_ACCESS_KEY_ID` The Access Key Id for an AWS credential.
++ `-aws_access_key_id=AWS_ACCESS_KEY_ID` The access key ID for an AWS credential.
   This credential is associated with an IAM user or role.
   This IAM user must have permissions to import images.
```

Comment:
can we:
1. enhance that key/secrect here must be the temp one generated from STS

Reply:
One more: hint them it will be visible for all project viewers

Reply:
Ok, then should we still mention that in the README? Maybe if the credentials will only be visible in the gcloud layer, it should be mentioned in the public doc instead.

Reply:
Correct, nothing to do here. Just do it for public doc and gcloud help text.